schema.graphql

```diff
@@ -433,14 +447,85 @@ type Match {
 }
 
 """
-Docs group: Match
+Docs group: Question
+"""
+type Packet {
+  """
+  A human-readable identifier. Often primarily numeric (e.g., `Packet 1`), but could also be identified by authors (`Chicago` or `Yale + Maryland`), as for packet-submission tournaments.
+  """
+  identifier: String!
+  
+  """
+  The packet number, if that notion is meaningful. May also be contained in the `identifier`, but need not be, e.g. if both `number` and `authors` are specified, the `identifier` will typically only contain one of them (whichever is more important).
+  """
+  number: Int
+  
+  """
+  The authors of the packet. May also be contained in the `identifier`, but need not be, e.g. if both `number` and `authors` are specified, the `identifier` will typically only contain one of them (whichever is more important).
+  """
+  authors: [String]
+
+  """
+  How the packet is expected to be used. An `extra` packet typically consists only of tossups, and is for use in overtime and/or as replacements, as necessary. Note that a packet might be used differently in case of unusual situations.
+  """
+  role: PacketRole
+
+  """
+  Any further details on the packet.
+  """
+  note: String
+
+  """
+  The contents of the packet, in order.
+  """
+  questions: [Question]
+}
+
+enum PacketRole {
+  regular
+  finals
+  extra
+  overtime
+  replacement
+  backup
+  tiebreaker
+  scrimmage
+}
+
+"""
+Docs group: Question
```

Comment:
I'd leave this in the `Match` group, but I don't feel super strongly about that.

Reply:
I waffled on this. I changed it to your version. Note, though, that there are now only two classes in `Question`.